site/src/examples/low-barrel/index.js

```diff
@@ -206,6 +206,18 @@
                     ]);
                 }
                 return data;
+            })
+            .decorate(function(sel) {
+                var height = Math.abs(chart.yScale().range()[0] - chart.yScale().range()[1]);
+                sel.enter()
+                    .selectAll('.resize.e>rect, .resize.w>rect')
+                    .style('visibility', 'visible')
+                    .attr('y', height / 4)
+                    .attr('rx', 4)
+                    .attr('ry', 4);
+
+                sel.selectAll('.resize.e>rect, .resize.w>rect')
```

Comment:
As a [y scale is set](https://github.com/mbostock/d3/blob/5b981a18db32938206b3579248c47205ecc94123/src/svg/brush.js#L81-L85) on the brush ([multi does this](https://github.com/ScottLogic/d3fc/blob/6cafee1565501c7eab8a254a8f53d4f2ddd0b327/src/series/multi.js#L51)), D3's brush component [resets the height of the `rect`](https://github.com/mbostock/d3/blob/5b981a18db32938206b3579248c47205ecc94123/src/svg/brush.js#L142) on every redraw, so it has to overridden within the update selection, rather than the enter selection.